src/renderer/components/inputs/elements/TabList.tsx

```diff
@@ -42,7 +43,7 @@ export const TabList = memo(({ value, onChange, reset, isDisabled, options }: Ta
             pt={1}
             size="sm"
             variant="unstyled"
-            onChange={handleChange}
+            onChange={() => {}} // Placeholder, no operation here
```

Comment:
Please use [our `noop` function](https://github.com/chaiNNer-org/chaiNNer/blob/7fbc9972ad318149f017d64e1ffb2528aa3a0411/src/common/util.ts#L11). It's slightly more efficient to use the same function instead of creating a new one each time, because it allows React to (potentially) do less work.

Reply:
Does this even actually need to be set? Could the onChange prop just be removed altogether?

Reply:
@RunDevelopment Thank you for the notice! I have made the change to use noop. 

@joeyballentine I tried removing onChange and it seems like it functions correctly as well. Should I just go ahead and delete it?

Reply:
Joey's right, we can remove this, so please go ahead. I thought this field was important, but I was wrong.

Reply:
The change has been made.